.teamcity/OpenSourceProjects_Storybook/Project.kt

```diff
@@ -21,13 +21,12 @@ object Project : Project({
     buildType(OpenSourceProjects_Storybook_CliTestLatestCra)
     buildType(OpenSourceProjects_Storybook_Examples)
     buildType(OpenSourceProjects_Storybook_Danger)
-    buildType(OpenSourceProjects_Storybook_ReactNative)
+    buildType(OpenSourceProjects_Storybook_NativeSmokeTests)
     buildType(OpenSourceProjects_Storybook_Docs)
     buildType(OpenSourceProjects_Storybook_Build_2)
     buildType(OpenSourceProjects_Storybook_CliTest)
     buildType(OpenSourceProjects_Storybook_Test)
     buildType(OpenSourceProjects_Storybook_Lint)
-    buildType(OpenSourceProjects_Storybook_Lint_Warnings)
```

Comment:
Why?

Reply:
Didn't realize we wanted both Lint giving warnings and error. Any reason behind that?

Reply:
"Lint" blocks merging PRs, and "Lint Warnings" doesn't. We don't want to block PR because of wrong indentation, but we want to fix it at some point afterwards. And to fix the issue, we need to know about it.

Reply:
Wouldn't that be achieved by removing "Lint" from "Build" and make it optional?

Reply:
Eslint doesn't only checks formatting. We still want everything that we marked as "error" to be a blocker